### DISCUSSION TITLE
Tell me how to change vartical line color of DataGrid header.

### DISCUSSION THREAD

**Discussion by maguro1kan** [AvaloniaUI/Avalonia#10521] [Q&A] [answered]
Hi, I have a question. Tell me how to change vartical line color of DataGrid header.
![image](https://user-images.githubusercontent.com/30754124/222169417-9dc83d0a-1316-456a-88e4-a7be75b10fe3.png)

Thanks

**Comment by timunie** [comment, marked as answer]
Which Avalonia version? 

------

Note: Basically you can look up the original styles in the source. There you should see the used values or resources. Most of the time you can simply override the specific brush, but rembmer to switch to the branch you use.

**Reply by maguro1kan** [reply]
Using 0.10.18. But I can do update to 11.0.0-preview.

**Reply by timunie** [reply]
In that case I think this line needs to be adjusted: https://github.com/AvaloniaUI/Avalonia/blob/401bc521d2947bf7699a530f3d6e43e7a32c6baf/src/Avalonia.Controls.DataGrid/Themes/Fluent.xaml#L182

**Reply by maguro1kan** [reply]
I see. It seems difficult for me.
Can this be easily changed in a future update?

**Reply by timunie** [reply]
What is difficult here?

Option 1. Override the Brush
```xml
<App.Resouces>
   <SolidColorBrush x:Key="DataGridGridLinesBrush">Red</SolidColorBrush>
</App.Resources>
```

Option 2: Create a  Style

```xml
<App.Styles>
  <Style Selector="DataGridColumnHeader">
    <Setter Property="SeparatorBrush" Value="Red" />
  </Style>
</App.Styles>
```

**Reply by maguro1kan** [reply]
Great! I did it. Thanks!
